src/Test/Perf/.gitignore

```diff
@@ -2,5 +2,7 @@ artifacts/
 infra/bin/
 results/
 temp/
+DataBackup*/
+consumptionTempResults.xml
```

Comment:
Assuming that these are generated on a per-run basis, they should go in src/Test/Perf/temp.